libs/MotionKit/include/MotionKit.h

```diff
@@ -51,10 +53,15 @@ class MotionKit
 	bool _stabilisation_requested  = false;
 	bool _rotate_x_turns_requested = false;
 
-	const float kReferenceAngle		   = 180.F;
-	const float kMinimalViableRobotPwm = 0.15F;	  // ? Under this pwm value, torque is too low to spin the wheel
+	const float kReferenceAngle = 180.F;
+	const float kPIDMaxValue	= 1.8F;
+
+	// ? Under this pwm value, torque is too low to spin the wheel without inertia.
+	// ? This value needs to be the lowest possible to prevent inertia drift after reaching target
```

Comment:
```suggestion
	// ? When the motor is stopped, PWM values under kMinimalViableRobotPwm are too low to generate enough torque for the motor to start spinning
	// ? At the same time, kMinimalViableRobotPwm needs to be the lowest possible to avoid overshooting when the target is reached
```